web/content/docs/benchmarks/reactive-transport/calcite.pandoc

```diff
@@ -31,7 +31,7 @@ The operator-splitting (OS) model formulation (Strang, 1968) with a sequential n
 
 A comparison of the numerical data is shown in the figure below. The numerical results of OGS-6 coincide with those of OGS-5.
```

Comment:
Another useful information would be:

"... coincide with those of OGS-5 within 25% relative error."

Adjust the numbers as needed.

Reply:
Added absolute error information.